src/openms/source/ANALYSIS/OPENSWATH/TransitionTSVFile.cpp

```diff
@@ -1038,22 +1038,28 @@ namespace OpenMS
 
     // Try to parse full UniMod string including modifications. If we fail, we
     // can force reading and only parse the "naked" sequence.
+    // Note: If the user did not provide a modified sequence string, we will
+    // fall back to the "naked" sequence by default.
     std::vector<TargetedExperiment::Peptide::Modification> mods;
     AASequence aa_sequence;
+    String sequence = tr_it->FullPeptideName;
+    if (sequence.empty()) sequence = tr_it->PeptideSequence;
```

Comment:
missing UniMod peptide sequence will now mean that the naked sequence is used instead by default and no `force_invalid_mods` is required

Reply:
👍